tests/testthat/test-reporter-junit.R

```diff
@@ -0,0 +1,27 @@
+context("JUnitReporter")
+
+
+test_that("Junit reporter regression", {
```

Comment:
Do you think this is needed as well as the saved xml file? It seems a bit redundant to me.